spec.bs

```diff
@@ -4004,36 +4042,64 @@ from querying the server during an auction.
 </div>
 
 <div algorithm>
- To <dfn>compute the key hash of ad</dfn> given an [=interest group=] |ig| and an [=interest group ad=] |igAd|:
-   1. Let |keyString| be the [=k-anonymity key=] formed from the [=string/concatenation=] of the following strings separated with U+000A LF:
+  To <dfn>compute the key hash of ad</dfn> given an [=interest group=] |ig| and an [=interest group ad=] |igAd|:
+    1. Let |keyString| be the [=k-anonymity key=] formed from the [=string/concatenation=] of the following strings separated with U+000A LF:
       * "AdBid"
       * the [=serialization of an origin|serialization=] of |ig|'s [=interest group/owner=]
       * the [=URL serializer|serialization=] of |ig|'s [=interest group/bidding url=]
       * the [=URL serializer|serialization=] of |igAd|'s [=interest group ad/render url=].
-   1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
+    1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
 </div>
 
 <div algorithm>
-  To <dfn>compute the key hash of reporting ID</dfn> given an [=interest group=] |ig| and an
-  [=interest group ad=] |igAd|:
-    1. Let |keyString| be a [=k-anonymity key=] formed from the [=string/concatenation=] of the following strings separated with U+000A (LF):
+  To <dfn>compute the key part for one of multiple reporting ids</dfn> given [=string=]-or-null |reportingId|:

```

Comment:
can you give me the code link? I'd like to check its implementation, specifically on the separator of concatenation.

Reply:
Sure [here it is]!(https://crsrc.org/c/third_party/blink/common/interest_group/interest_group.cc;drc=451093bbaf7fe812bf67d27d760f3bb64c92830b;l=127)